exonum-java-binding-qa-service/src/main/java/com/exonum/binding/qaservice/QaServiceImpl.java

```diff
@@ -96,7 +96,7 @@ protected Schema createDataSchema(View view) {
     List<HashCode> stateHashes = super.getStateHashes(snapshot);
     // Log the state hashes, so that the values passed to the native part of the framework
     // are known.
-    logger.error("state hashes: {}", stateHashes);
+    logger.debug("state hashes: {}", stateHashes);
```

Comment:
Btw, it was one of the requirements of QA service (you can easily find it in wiki), not sure the reason though

Reply:
Thanks, that's exactly the reason. If there isn't a comment clarifying that, please add.

Reply:
Reverted this change. What is the reason for the `error` level requirement? Seeing errors in the log confuses me